en/docs/release-notes/version-3.3.0.md

```diff
@@ -21,22 +21,22 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
 ## Integrated GraphQL Tryit
 ![graphql-tryit](../../img/release-notes/graphql-tryit.gif)
 
-- You can try out the GraphQL services using the `Tryit` codelens 
+- You can try out the GraphQL services using the `Tryit` CodeLens 
 - You can send and test any request with headers to your GraphQL service in VS Code itself without using any third-party tools
 - The GraphQL explorer will help you to explore the available endpoints and generate the query for you
 
-## Record editor
+## Improved Record Editor
```

Comment:
Also I don't think `improved` matches here. All the other sub titles are just the feature/component name while this has a verb.

Reply:
Fixed in https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/pull/24.